diff --git a/dist/DtsWorker-Bt7-_SGH.d.ts b/dist/DtsWorker-Bt7-_SGH.d.ts
index 420818771e5e6f6fec9a3e649eaec3976bca939c..db2093f208feecfb432ca17e702b318222568125 100644
--- a/dist/DtsWorker-Bt7-_SGH.d.ts
+++ b/dist/DtsWorker-Bt7-_SGH.d.ts
@@ -1,3 +1,3 @@
 import { moduleFederationPlugin } from "@module-federation/sdk";
-import ts from "typescript";
+import ts from "typescript-mf";
 import { ChildProcess } from "child_process";
diff --git a/dist/esm/expose-rpc-CDdqA9PX.mjs b/dist/esm/expose-rpc-CDdqA9PX.mjs
index 8bbf1d6df2c152bf07c5ab501d0e3ccf000e8531..ae8d7be29d690da82a08a7ba751da43d847244fd 100644
--- a/dist/esm/expose-rpc-CDdqA9PX.mjs
+++ b/dist/esm/expose-rpc-CDdqA9PX.mjs
@@ -6,7 +6,7 @@ import { fileURLToPath } from "url";
 import path, { dirname, extname, isAbsolute, join, normalize, relative, resolve, sep } from "path";
 import { cp, mkdir, readFile, readdir, rm, stat, writeFile } from "fs/promises";
 import { utils } from "@module-federation/managers";
-import typescript from "typescript";
+import typescript from "typescript-mf";
 import { styleText } from "node:util";
 import { ENCODE_NAME_PREFIX, MANIFEST_EXT, TEMP_DIR, decodeName, getProcessEnv, inferAutoPublicPath, parseEntry } from "@module-federation/sdk";
 import { Agent } from "undici";
diff --git a/dist/expose-rpc-BpRiu3VK.js b/dist/expose-rpc-BpRiu3VK.js
index f0eedc2b3afbf36c30d2ec226112742c19320cca..5d78fd870b3ee257e083a980fd06a3bd99075259 100644
--- a/dist/expose-rpc-BpRiu3VK.js
+++ b/dist/expose-rpc-BpRiu3VK.js
@@ -7,7 +7,7 @@ let path = require("path");
 path = require_Action.__toESM(path);
 let fs_promises = require("fs/promises");
 let _module_federation_managers = require("@module-federation/managers");
-let typescript = require("typescript");
+let typescript = require("typescript-mf");
 typescript = require_Action.__toESM(typescript);
 let node_util = require("node:util");
 let _module_federation_sdk = require("@module-federation/sdk");
