diff --git a/dist/981.js b/dist/981.js
index 6e3141ac7a6c0390acc735c19fb670c2f3ff4a91..1dd5f859fdece3486edd50d3ff429bb94e6a5fb5 100644
--- a/dist/981.js
+++ b/dist/981.js
@@ -3427,7 +3427,7 @@ const loadAstGrepNapi = ()=>{
     if (!astGrepNapi) astGrepNapi = utils_require('@ast-grep/napi');
     return astGrepNapi;
 };
-const ts = utils_require("typescript");
+const ts = utils_require("typescript-rslib");
 const createStyler = (style)=>(text)=>styleText(style, String(text));
 const color = {
     bold: createStyler('bold'),
